scripts/front/add-missing-keys-to-other-languages.js

```diff
@@ -0,0 +1,66 @@
+// Add missing keys to non-english languages from `en.json`.
+// This script eases the process of translating strapi to other languages.
+// Usage:
+//   node scripts/front/add-missing-keys-to-other-languages.js [language]
+// Example:
+//   node scripts/front/add-missing-keys-to-other-languages.js vi
+'use strict';
+
+const { join, dirname } = require('path');
+const { promisify } = require('util');
+const fs = require('fs-extra');
+const glob = promisify(require('glob').glob);
+
+const addMissingKeyForSingleFile = async filePath => {
+  console.log('Start adding missing keys to', filePath);
+  try {
+    // Read translation file
+    const currentTranslationFileJSON = await fs.readJSON(filePath);
+    // Read en.json
+    const mainTranslationFile = join(dirname(filePath), 'en.json');
+    const mainTranslationFileJSON = await fs.readJSON(mainTranslationFile);
+    // Add missing keys from en.json to translation file
+    const updatedFile = Object.keys(mainTranslationFileJSON).reduce((acc, current) => {
+      if (currentTranslationFileJSON[current]) {
+        acc[current] = currentTranslationFileJSON[current];
+      } else {
+        acc[current] = mainTranslationFileJSON[current];
+      }
+
+      return acc;
+    }, {});
+    await fs.writeJson(filePath, updatedFile, { spaces: 2 });
+    console.log('Added missing keys to ', filePath);
```

Comment:
I suggest removing space to add consistency
```suggestion
    console.log('Added missing keys to', filePath);
```